cloner_test.go

```diff
@@ -123,6 +123,16 @@ var _ = Describe("GitStore", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(files).To(HaveLen(0))
 			})
+
+			It("should be able to do it multiple times", func() {
+				rs := NewRepoStore(tmpDir)
```

Comment:
I think you will need a new `rs` for each test, since the `rs` holds some locks in memory and a map of which repos are being fetched, does this test fail if you take out the `if` statement in `cloner.go`?

Reply:
It does fail, but that's because it's initially cloned in the `BeforeEach`. Was literally looking at this before I pushed and was wondering the same thing and it only clicked just now. I'll move it into the loop so that it's more clear.